sql/log4j.properties

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
This is used - https://github.com/apache/spark/pull/18702/files#diff-a4b1e8e0e72fd59bd246285a34b21a45R49

I hesitated to add this one but just added to make the infos quiet when a spark session is initialised.

There is a similar one in R - https://github.com/apache/spark/blob/master/R/run-tests.sh#L26 and https://github.com/apache/spark/blob/master/R/log4j.properties but this case logs are quite small though.

```bash
$ sh create-docs.sh
```

**Before**

```
Generating markdown files for SQL documentation.
Generating HTML files for SQL documentation.
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /.../spark/sql/site
```

**After**

```
Generating markdown files for SQL documentation.
Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
17/07/25 18:02:57 INFO SparkContext: Running Spark version 2.3.0-SNAPSHOT
17/07/25 18:02:57 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
17/07/25 18:02:57 INFO SparkContext: Submitted application: GenSQLDocs
17/07/25 18:02:57 INFO SecurityManager: Changing view acls to: hyukjinkwon
17/07/25 18:02:57 INFO SecurityManager: Changing modify acls to: hyukjinkwon
17/07/25 18:02:57 INFO SecurityManager: Changing view acls groups to:
17/07/25 18:02:57 INFO SecurityManager: Changing modify acls groups to:
17/07/25 18:02:57 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(hyukjinkwon); groups with view permissions: Set(); users  with modify permissions: Set(hyukjinkwon); groups with modify permissions: Set()
17/07/25 18:02:58 INFO Utils: Successfully started service 'sparkDriver' on port 62695.
17/07/25 18:02:58 INFO SparkEnv: Registering MapOutputTracker
17/07/25 18:02:58 INFO SparkEnv: Registering BlockManagerMaster
17/07/25 18:02:58 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
17/07/25 18:02:58 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
17/07/25 18:02:58 INFO DiskBlockManager: Created local directory at /private/var/folders/9j/gf_c342d7d150mwrxvkqnc180000gn/T/blockmgr-13909519-0864-4aa3-82fb-eba4b9d1e527
17/07/25 18:02:58 INFO MemoryStore: MemoryStore started with capacity 366.3 MB
17/07/25 18:02:58 INFO SparkEnv: Registering OutputCommitCoordinator
17/07/25 18:02:58 INFO Utils: Successfully started service 'SparkUI' on port 4040.
17/07/25 18:02:58 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://192.168.0.146:4040
17/07/25 18:02:58 INFO SparkContext: Added file file:/.../spark/sql/gen-sql-markdown.py at file:/.../spark/sql/gen-sql-markdown.py with timestamp 1500973378636
17/07/25 18:02:58 INFO Utils: Copying /.../spark/sql/gen-sql-markdown.py to /private/var/folders/9j/gf_c342d7d150mwrxvkqnc180000gn/T/spark-0eb99795-6a5e-4101-8ed2-c55b3ba80173/userFiles-64210993-0ed8-4c05-85bd-83d13ae22831/gen-sql-markdown.py
17/07/25 18:02:58 INFO Executor: Starting executor ID driver on host localhost
17/07/25 18:02:58 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 62696.
17/07/25 18:02:58 INFO NettyBlockTransferService: Server created on 192.168.0.146:62696
17/07/25 18:02:58 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
17/07/25 18:02:58 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 192.168.0.146, 62696, None)
17/07/25 18:02:58 INFO BlockManagerMasterEndpoint: Registering block manager 192.168.0.146:62696 with 366.3 MB RAM, BlockManagerId(driver, 192.168.0.146, 62696, None)
17/07/25 18:02:58 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 192.168.0.146, 62696, None)
17/07/25 18:02:58 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 192.168.0.146, 62696, None)
17/07/25 18:02:59 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir ('/.../spark/sql/_spark-warehouse').
17/07/25 18:02:59 INFO SharedState: Warehouse path is '/.../spark/sql/_spark-warehouse'.
17/07/25 18:02:59 INFO StateStoreCoordinatorRef: Registered StateStoreCoordinator endpoint
17/07/25 18:02:59 INFO SparkUI: Stopped Spark web UI at http://192.168.0.146:4040
17/07/25 18:02:59 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
17/07/25 18:02:59 INFO MemoryStore: MemoryStore cleared
17/07/25 18:02:59 INFO BlockManager: BlockManager stopped
17/07/25 18:02:59 INFO BlockManagerMaster: BlockManagerMaster stopped
17/07/25 18:02:59 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
17/07/25 18:02:59 INFO SparkContext: Successfully stopped SparkContext
17/07/25 18:02:59 INFO ShutdownHookManager: Shutdown hook called
17/07/25 18:02:59 INFO ShutdownHookManager: Deleting directory /private/var/folders/9j/gf_c342d7d150mwrxvkqnc180000gn/T/spark-0eb99795-6a5e-4101-8ed2-c55b3ba80173
17/07/25 18:02:59 INFO ShutdownHookManager: Deleting directory /private/var/folders/9j/gf_c342d7d150mwrxvkqnc180000gn/T/spark-0eb99795-6a5e-4101-8ed2-c55b3ba80173/pyspark-4cdacd99-6ab2-4a1e-9b76-c4d6d457477e
Generating HTML files for SQL documentation.
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /.../spark/sql/site
```

Would you prefer to get rid of this?